whotracksme/data/Readme.md

```diff
@@ -1,4 +1,4 @@
-# WhoTracks.me Data
+# Data
```

Comment:
Did you mean to change the page title?

Reply:
Yes but that's optional. I proposed it because there are 2 main data sources: SQL database and CSVs with tracking metrics. 

I realize you refer to "WhoTracks.me data" as both sources but this was confusing for me at the first glance.
Proposed solution would be to use 2 names: SQL database and WhoTracks.me datasets (CSVs) but I leave that up to you guys.